eng/service.proj

```diff
@@ -9,14 +9,15 @@
 
   <ItemGroup>
     <ExcludeMgmtLib Include="..\sdk\*\*.Management.*\**\*.csproj;..\sdk\*mgmt*\**\*.csproj;" />
+    <ExcludeTrack1Projects Include="..\sdk\*\Microsoft.*\**\*.csproj" Condition="'$(ExcludeTrack1)'=='true'" />
```

Comment:
Why do you need to exclude track1?

Reply:
Because it doesn't work with our current test infrastructure. Internal conversations with Alex, Ted, and Bruce confirm we don't need to run track 1 for Search. In fact, in general, if they weren't already running we probably don't want to run them if no feature changes are planned (conversation with Scott K.).

If we want to make them work in the future (I will open a bug in the backlog, but we may just end up closing it) more work is necessary and not worth the effort now.